new-lamassu-admin/src/pages/Transactions/Address.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react'
+import classnames from 'classnames'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+import { replace } from 'lodash/fp'
+import { makeStyles } from '@material-ui/core/styles'
+
+import { addressStyles } from './Transactions.styles'
+
+import Popover from '../../components/Popper'
+import { ReactComponent as CopyIcon } from '../../styling/icons/action/copy/copy.svg'
+import { comet } from '../../styling/variables'
+
+const Address = ({ className, type, children, ...props }) => {
+  const [anchorEl, setAnchorEl] = useState(null)
+
+  useEffect(() => {
+    setTimeout(() => setAnchorEl(null), 3000)
+  }, [anchorEl])
```

Comment:
Why is this necessary?

Reply:
This auto closes the 'Copied to clipboard!' popper after 3s. The Material-Ui Popper uses a toggle anchor, so the user would have to click the Copy button again to close the popper. I think it's also a nice touch.